src/sync.h

```diff
@@ -148,72 +148,74 @@ inline void AssertLockNotHeldInline(const char* name, const char* file, int line
 #define AssertLockNotHeld(cs) AssertLockNotHeldInline(#cs, __FILE__, __LINE__, &cs)
 
 /** Wrapper around std::unique_lock style lock for Mutex. */
-template <typename Mutex, typename Base = typename Mutex::UniqueLock>
-class SCOPED_LOCKABLE UniqueLock : public Base
+template <typename MutexType>
+class SCOPED_LOCKABLE DebugLock : public MutexType::UniqueLock
 {
 private:
+    using PARENT = typename MutexType::UniqueLock;
```

Comment:
I don't think we need to unify the names. Base seems fine to keep as is

Reply:
Changed to `Base`.